src/main/java/nl/novi/techiteasy/security/MyUserDetailsService.java

```diff
@@ -0,0 +1,28 @@
+package nl.novi.techiteasy.security;
+
+import nl.novi.techiteasy.models.User;
+import nl.novi.techiteasy.repositories.UserRepository;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+
+import java.util.Optional;
+
+public class MyUserDetailsService implements UserDetailsService {
+    private final UserRepository userRepository;
+
+    public MyUserDetailsService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+    @Override
+    public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
+        Optional<User> ou = userRepository.findById(username);
```

Comment:
Nogmaals dezelfde opmerking: ik vind "ou" een nietszeggende naam. Ik kan aan de naam onmogelijk ontleden wat voor een variabele het is.